app/assets/stylesheets/_category.scss

```diff
@@ -24,3 +24,18 @@
   color: #bfa24b;
   text-decoration: none;
 }
+
+#menu .floaters{
+  width:18%;
+  border:1px solid white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Colon after property should be followed by one space